### ISSUE TITLE
FR: requestable_data()

### ISSUE THREAD

**Issue by aaronpeikert** [cjvanlissa/worcs#95]
If researchers can not / do not want to share their data, but feel uncomfortable with synthesizing their data I think it would be great if they could leave a note about how to obtain the real data. So researchers could leave a note like "Please write a mail to researcher@uni.edu with a request for data following these guideline: uni.edu/psy/request-data".
Related to #94

**Comment by cjvanlissa** [comment]
@aaronpeikert `closed_data()` now adds a paragraph to the readme that mentions requesting the data. Do you think it's sufficient?

**Comment by aaronpeikert** [comment]
Yes! I prefer data encryption anyway for these cases (if we can figure out a save way to handle the encryption keys). BTW: I wasn't able to test this feature on branches master or ds_revision.
